scripts/o3de/o3de/enable_gem.py

```diff
@@ -96,17 +100,45 @@ def enable_gem_in_project(gem_name: str = None,
         if not project_enabled_gem_file.is_file():
             project_enabled_gem_file.touch()
 
-    # Before adding the gem_dependency check if the project is registered in either the project or engine manifest
+    # Before adding the gem_dependency check if the gem is registered in either the project or engine manifest
     buildable_gems = manifest.get_engine_gems()
     buildable_gems.extend(manifest.get_project_gems(project_path))
     # Convert each path to pathlib.Path object and filter out duplicates using dict.fromkeys
     buildable_gems = list(dict.fromkeys(map(lambda gem_path_string: pathlib.Path(gem_path_string), buildable_gems)))
 
+    # check compatibility
+    if force:
+        logger.warning(f'Bypassing version compatibility check for {gem_json_data["gem_name"]}.')
```

Comment:
IMHO, if 'force' was explicitly passed in, then this should not be a warning but an info